packages/components/src/Tabs/Tabs.tsx

```diff
@@ -52,7 +52,7 @@ export function useTabs(props?: UseTabsProps) {
         setSelectedIndex(index)
       }
     },
-    selectedIndex,
+    selectedIndex: isControlled ? props?.controlledIndex : selectedIndex,
```

Comment:
```suggestion
    selectedIndex: isControlled && props ? props.controlledIndex : selectedIndex,
```
I'm having second thoughts about this fancy TS syntax. Let's just go old-school to be safe.